packages/components/src/organisms/RatingSummary/RatingDistributionItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { forwardRef, type HTMLAttributes } from 'react'
+import Icon from '../../atoms/Icon'
+
+export interface RatingDistributionItemProps
+  extends HTMLAttributes<HTMLLIElement> {
+  /**
+   * Star rating index. It should be a number between 1 and 5.
+   */
+  ratingIndex: number
+  /**
+   * Percentage value. It should be a number between 0 and 100.
+   */
+  value: number
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const ItemStar = ({ ratingIndex }: { ratingIndex: number }) => (
+  <span data-fs-rating-distribution-item-star>
+    <p>{ratingIndex}</p>
+    <Icon data-fs-rating-distribution-item-star-icon name="Star" />
+  </span>
+)
+
+const ItemBar = ({ value }: { value: number }) => (
+  <div data-fs-rating-distribution-item-progress-bar>
+    <div data-fs-rating-distribution-item-progress-bar-track>
+      <div
+        data-fs-rating-distribution-item-progress-bar-fill
```

Comment:
todo: could you also change those data attributes to not use the "progress bar" term?